scale/job/test/test_views.py

```diff
@@ -362,7 +362,7 @@ def test_successful(self):
         """Tests successfully calling POST jobs view to queue a new job"""
 
         json_data = { 
-            "input" : {},
+            "job_data" : {},
```

Comment:
shouldn't it be sending input and configuration (with configuration being ignored for now)?